obs.py

```diff
@@ -666,6 +675,70 @@ def send_to_aws(self):
             else:
                 time.sleep(0.2)
 
+    # Note this is a thread!
+    def fast_to_aws(self):
```

Comment:
I see now what you mean by better ways of doing things! But I agree it's important to get the fast queuing working first and nicer later. This is fine as a fix for the time being, and I have ideas on how we can improve it in a later PR if you would like.